GitHub-Test-4-PyNLPDiscuss/rorange.md

```diff
@@ -0,0 +1,23 @@
+#Remi's Git Test 4 Submission
+
+1.) I found the *Entity Ruler* interesting because it doesn't branch too far away from what we're discussing and allows us to take a little more control over the code we're working with. It feels like bending the code to do what I want it to do.
+
+ 
+
+2.) As before, I think the Entity Ruler is very neat. It could be used to assign "magical" items in fiction books than going down the path of BookNLP. It feels like a good learning point.
+
+ 
+
+3.) http://www.script-o-rama.com/movie_scripts/t/top-gun-script-transcript-cruise.html Links to an external site.Not really useful but I think it'd be fun to go through old 80s movies scripts and find slang, references, and songs that aren't picked up by modern verbage in code.
```

Comment:
I do agree that it would be interesting to go through 80s movies like Top Gun to find quirks, slang, etc.